sycl/source/detail/pi.cpp

```diff
@@ -74,7 +117,7 @@ void initialize() {
 //       but for now it is useful to see every failure.
 //
 [[noreturn]] void die(const char *Message) {
-  std::cerr << "pi_die: " << Message << std::endl;
+  std::cout << "pi_die: " << Message << std::endl;
```

Comment:
Is it not what `std::cerr` is made for?

Reply:
yes. Changed it by mistake. thank you for noticing it.